lib/login.ts

```diff
@@ -14,10 +14,20 @@ export class UserDataStore implements IUserDataStore {
 	private cookies: IStringDictionary;
 	private user: any;
 
+	private static APPBUILDER_EDITION_TYPE: IStringDictionary = {
```

Comment:
Unfortunately no :(
We receive only the enum from the Server (https://github.com/Icenium/Ice/blob/master/Common/Common.Data/Authentication/EditionType.cs) while we have to send the "Description" of each item. Same logic is implemented in Mist:
https://github.com/Icenium/Ice/pull/3854/files